### PR TITLE
[Ingest] Remove the fact that ingest was a plugin from the docs.

### PR DESCRIPTION
**Pull request by martijnvg**

### CODE REVIEW

File: docs/reference/ingest/ingest.asciidoc [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+== Ingest
 
-The ingest plugin can be used to pre-process documents before the actual indexing takes place.
-This pre-processing happens by the ingest plugin that intercepts bulk and index requests, applies the
+Ingest can be used to pre-process documents before the actual indexing takes place.
+This pre-processing happens by Ingest that intercepts bulk and index requests, applies the
 transformations and then passes the documents back to the index or bulk APIs.
 
-The ingest plugin is disabled by default. In order to enable the ingest plugin the following
+Ingest is enabled by default. In order to disable ingest the following
```
Comment (javanna) [inline docs/reference/ingest/ingest.asciidoc]: we should rather talk about this as a new node type more than saying ingest enabled like we did when it was a plugin?

File: docs/reference/ingest/ingest.asciidoc [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 
 [source,yaml]
 --------------------------------------------------
-node.ingest: true
+node.ingest: false
 --------------------------------------------------
 
-The ingest plugin can be installed and enabled on any node. It is possible to run ingest
-on an master and or data node or have dedicated client nodes that run with ingest.
+It is possible to enable ingest on an master and or data node or have dedicated client nodes that run with ingest.
```
Comment (javanna) [inline docs/reference/ingest/ingest.asciidoc]: on any node?

**Comment by javanna** [comment]
thanks @martijnvg left a few minor comments, LGTM otherwise

File: docs/reference/ingest/ingest.asciidoc (hunk 2 above)
Comment (martijnvg) [inline docs/reference/ingest/ingest.asciidoc]: does the last commit makes sense?

**Comment by javanna** [comment]
++ thanks @martijnvg

**Comment by talevy** [comment]
thanks! missed that

**Pull request merged**